README.md

```diff
@@ -26,28 +26,79 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
 
 ## Installation
 
-1. Install phpcs:
+### Composer
 
-        pear install PHP_CodeSniffer
+Using [Composer](https://getcomposer.org/) is the preferred way.
 
-2. Find your PEAR directory:
+1. Add the MO4 repository
 
-        pear config-show | grep php_dir
+        composer config repositories.mo4 vcs https://github.com/mayflower/mo4-coding-standard.git
 
-3. Copy, symlink or check out the Symfony coding standard and this repository to their respecting folders inside the
-   phpcs `Standards` directory:
+If composer complains about a missing composer.json, just create an empty one
 
-        cd /path/to/pear/PHP/CodeSniffer/src/Standards
-        git clone https://github.com/djoos/Symfony-coding-standard.git Symfony
-        mv Symfony/Symfony/* Symfony/
-        git clone https://github.com/mayflower/mo4-coding-standard.git MO4
+        echo '{}' > composer.json
 
-4. Select the MO4 ruleset as your default coding standard:
+2. Add the MO4 coding standard to `composer.json`
 
-        phpcs --config-set default_standard MO4
+        composer require --dev mayflower/mo4-coding-standard:dev-master
+
+3. Optionally, you might set MO4 as default coding standard
+
+        ./vendor/bin/phpcs --config-set default_standard MO4
+
+4. Profit
+
+        ./vendor/bin/phpcs path/to/my/file.php
+
+or
```

Comment:
the alignment in the rendered document (at least when viewing on GH) is off here and in two similar occurrences